wp-littlefoot.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Plugin Name:       littlefoot
+ * Description:       Footnotes without the footprint.
+ * Plugin URI:        https://github.com/s3rgiosan/wp-littlefoot
+ * Requires at least: 6.1
+ * Requires PHP:      7.4
+ * Version:           0.1.0
+ * Author:            Sérgio Santos
+ * Author URI:        https://s3rgiosan.dev/?utm_source=wp-plugins&utm_medium=wp-littlefoot&utm_campaign=author-uri
+ * License:           MIT
+ * License URI:       https://opensource.org/license/mit/
```

Comment:
@goblindegook I kept the same license from littlefoot

Reply:
And can you keep the same license, given it's a WordPress plugin with WordPress code mixed together with rest?

Reply:
Gray area? 🤷‍♂️ Really don't know. I'm not planning to release this in wp.org so probably not _a thing_, but https://opensource.stackexchange.com/questions/8520/can-i-publish-a-wordpress-site-as-mit